app/src/main/java/ip/team13/petowner/helpers/AppConstants.kt

```diff
@@ -0,0 +1,7 @@
+package ip.team13.petowner.helpers
+
+object AppConstants {
+    object SharedPrefKeys {
+        const val ARG_FCM_TOKEN = "ARG_FCM_TOKEN"
```

Comment:
Is there any benefit on using `object`s to hold `const val`s as oposed to declaring them directly inside a Kotlin file? It feels like we're occupying some extra memory that is not really needed to achieve what we want

Reply:
Just for better structure and ease of access when coding, from my perspective.